torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1466,6 +1476,10 @@ def method_tests():
         ('ceil', (), NO_ARGS, 'scalar', (True,)),
         ('rad2deg', (S, S, S), NO_ARGS),
         ('deg2rad', (S, S, S), NO_ARGS),
+        # Removing the 'rsqrt' entries leads to failure in
+        # test_index_fill_variable_dim_*
+        # TODO: Remove when fixed.
+        # Reference: https://github.com/pytorch/pytorch/issues/48230
```

Comment:
Weird Bug!

Reply:
That is crazy! I bumped its priority.

Reply:
hmm. I wonder whether this is possibly related: that `torch/testing/_internal` is packaged with the pytorch distribution (hence packaged in build time) and the test folder is not? 

I sometimes get messed up test results when switching between git branches (python only changes) without recompiling